src/Ninject.Test/Integration/ReadOnlyKernelTests.cs

```diff
@@ -81,6 +81,38 @@ public void TryGetOfT_Parameters_ReturnsNullIfMultipleBindingsExistForADependenc
                 bindings.Should().HaveCount(1);
             }
 
+            [Fact]
+            public void TryGetOfT_NameAndParameters_ReturnsNullIfMultipleBindingsExistForResolvedService()
```

Comment:
The two unit tests for the generic `TryGet<T>` methods were already successful before the change in this PR